pkg/runtime/nodes.go

```diff
@@ -38,6 +39,8 @@ const (
 	RemoteAddRoute                  = "seautil route add --host %s --gateway %s"
 	RouteOK                         = "ok"
 	LvscareStaticPodCmd             = `echo "%s" > %s`
+	ReplaceImageSocket              = `sed '/KUBELET_EXTRA_ARGS/ s!$! --image-service-endpoint=/var/run/cri-resmgr/cri-resmgr.sock!' -i /etc/systemd/system/kubelet.service.d/10-kubeadm.conf `
```

Comment:
I guess replace cri-resmgr with sealer-cri-shim could be better. We are not using the complete cri-resmgr, are we?

Reply:
Yes, it's just a sock file name, we actually use sealer-cri-shim inside.